components/portal/timeout/controllers.js

```diff
@@ -138,15 +146,27 @@ define(['angular'], function(angular) {
         openFrom: 'left',
         closeTo: 'right',
         controller: function DialogController($scope, $mdDialog) {
-          $scope.closeDialog = function() {
+          $scope.continueBtn = function() {
+            if (isContinue) {
+              refreshSession();
+            } else {
+              reload();
+            }
             $mdDialog.hide();
           };
-          $scope.reloadPage = function() {
-            reload();
+          $scope.continueLabel = function() {
+            return (isContinue)?'OK':'Reload';
+          };
+          $scope.dialogText = function() {
+            var result = 'Your session will expire soon due to inactivity. ' +
+              'Please press OK to continue, otherwise you will be logged out.';
```

Comment:
Microcopy: Given that we're not going to log people out of the IdP, I wonder if text more parallel with the after-expiration text would better set user expectations, something like 'Press OK to continue, otherwise your session will expire.'  , Then it's telling them the session will expire, and then a minute later, it's, hey, the session expired.